example/main.go

```diff
@@ -4,17 +4,16 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/icza/bitio"
+	"github.com/woven-planet/go-zserio/ztype"
 	"myprojects.home/zserio-example/contacts"
 )
 
 func main() {
+	writer := ztype.NewWriter(os.Stdout)
+	defer writer.Close()
```

Comment:
Normally I would say we should catch and handle errors from `Close()`. Without that you could silently end up with truncated files. But since this is just an example I'm fine with skipping that.

Reply:
OK, Thanks. Just to continue the discussion, the only thing that we could do in this case is to notify the user?

Reply:
It should impact the exit-code as well.

Reply:
Reverted this.